queries/jsdoc/highlights.scm

```diff
@@ -19,6 +19,13 @@
 
 (identifier) @variable
 
+(tag
+  (identifier) @nospell)
+
+(tag
+  (optional_identifier
+    (identifier) @nospell))
+
```

Comment:
Maybe this should just be:
```suggestion
(identifier) @variable @nospell
```
that way it works with identifiers in `(path_expression)`, `(member_expression)`, etc.

Reply:
You are probably right. I changed it.